src/lnd-actions/lookup-invoice.js

```diff
@@ -4,14 +4,14 @@ const { deadline } = require('../grpc-utils')
  * Check's an invoice status
  *
  * @see http://api.lightning.community/#lookupInvoice
- * @param {string} paymentHash
+ * @param {Object} paymentHash
  * @param {Object} opts
  * @param {LndClient} opts.client
  * @returns {Promise<Object>}
  */
 function lookupInvoice (paymentHash, { client }) {
   return new Promise((resolve, reject) => {
-    client.lookupInvoice({ rHashStr: paymentHash }, { deadline: deadline() }, (err, res) => {
+    client.lookupInvoice(paymentHash, { deadline: deadline() }, (err, res) => {
```

Comment:
Might be more clear if we change the name to something like `hashObject` if we change the type